### ISSUE TITLE
Pattern synonyms do not support linear fields

### ISSUE THREAD

**Issue by ners** [haskus/packages#54]
```
src/tests/EADT.hs:33:1: error: [GHC-15172]
    • Pattern synonyms do not support linear fields (GHC #18806):
        a %1 -> EADT xs %1 -> EADT xs
    • In the declaration for pattern synonym ‘Cons’
   |
33 | eadtPattern 'ConsF "Cons"
   | ^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by hsyl20** [comment]
I've already fixed this in #53 / https://github.com/haskus/packages/commit/ced697b6b5aa70cedfef0b0fc292f9fab501f774

I don't remember if I've made a release though.

**Comment by hsyl20** [comment]
I've made a new release with the fix: https://hackage.haskell.org/package/haskus-utils-variant
